### DISCUSSION TITLE
panic(cpu 2 caller Oxffffff8006fe5ac4)

### DISCUSSION THREAD

**Discussion by digitaldiamont** [hackintosh-club/intel-nuc10#85] [General]
Hello, if I ask stupid questions, I ask for your understanding, I'm a beginner:
I did the GFG-Lock procedure,
then updated the bios and set it according to the instructions.
Then the ready-made EFI file in the EFI directory.
I have installed Monterey and Ventura, but I get a kernel panic when booting on both systems with every 2nd boot process.
I use the NUC10 i3.

Are further steps necessary for a reliable function?

Many Thanks

[panic-2022-11-15-161851.txt](https://github.com/hackintosh-efi/intel-nuc10/files/10014821/panic-2022-11-15-161851.txt)

**Comment by mrasong** [comment]
Try to check your memory, seems there are some thing wrong with your memory.

**Reply by digitaldiamont** [reply]
Thanks, I will try if I get other memory. 
But with Windows there are absolute no problems.

**Reply by mrasong** [reply]
> Thanks, I will try if I get other memory. But with Windows there are absolute no problems.

Did you ever got a stop error (AKA Blue Screen of Death) under windows?
You can use some tools checking you memory healthy. 
I got this error when I used another hackintosh, due to the bad memory error.

**Reply by digitaldiamont** [reply]
No, I never got a Blue Screen or any other problems with windows. 
Today I used the windows memory test  - no problems found. 
Could it be a GFG-Lock problem? I am not sure if I did everything right at this point.  
I also have the problem during MacOS installation. 
Have a nice day, thanks for your reply!

**Reply by mrasong** [reply]
Make sure your CGF-LOCK value is 0. 
If not, update your BIOS to 0055, use the OC 's cfg-lock tool, toggle the value to 0.

**Reply by digitaldiamont** [reply]
I came from BIOS 52, updated to 55 and changed the values to 0, than updated to BIOS 58 und checked if the values still to 0.
I was a little irritated that there were 2 CFG variables ​​here. Look at the screenshot.
I set both to value 0 - right?

![screenshot](https://user-images.githubusercontent.com/118297045/202542102-8341eafd-b6f7-4d2a-a497-8e0ecfc9ce93.jpg)

**Reply by digitaldiamont** [reply]
Unfortunately I haven't found a solution to this day. If the system is started, then it is stable. However, the kernel panics continue to exist when booting.
